next.config.default.js

```diff
@@ -0,0 +1,9 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+    reactStrictMode: true,
+    assetPrefix: 'https://verde.princeton.edu/pun/dev/pfclm-flask-app',
+    basePath: '/pun/dev/pfclm-flask-app',
```

Comment:
I think this should reflect the URL of the prod deployment (https://verde.princeton.edu/pun/sys/pfclm-flask) and the basePath of the prod environment (`/pun/sys/pfclm-flask`) instead of a dev app. 

This also introduces a slight issue with way the frontend repo does the copying of the static assets because the app is in a directory called `pfclm-flask` on Verde, not `pfclm-flask-app` (we had changed the name of the repo at some point and if we changed the name of the app now it would change the app's URL and break the CUAHSI connection).